ivy/functional/backends/torch/sub_backends/__init__.py

```diff
@@ -0,0 +1,18 @@
+import importlib
+
+import ivy
+
+_available_sub_backends = []
+_current_sub_backends = []
+
+
+def current_sub_backends():
+    return _current_sub_backends
+
+
+def available_sub_backends():
+    return _available_sub_backends
+
+
+if importlib.util.find_spec("xformers"):
```

Comment:
This logic should be done dynamically, in other words, this shouldn't be hardcoded in the init file, but rather discover installed subackends